Packs/Palo_Alto_Networks_WildFire/Playbooks/playbook-Detonate_File_-_WildFire.yml

```diff
@@ -284,20 +283,29 @@ tasks:
                 - - operator: match
                     left:
                       value:
-                        simple: File.Type
+                        simple: inputs.File.Type
```

Comment:
why did we need to add the `input`? did that even work before?

Reply:
@yuvalbenshalom I think it could have worked only if the PB context was set to global. but it is definitely a gap that was left.